src/part05.rs

```diff
@@ -47,7 +47,8 @@ impl BigInt {
         }
     }
 
-    // We can convert any vector of digits into a number, by removing trailing zeros. The `mut`
+    // Any vector of digits, which meets the structure of BigInt's `data` field, can be easily
```

Comment:
Okay, so maybe the wording should be something like "We can convert any little-endian vector of digits (i.e., least-significant digit first)" -- would that have helped?